pkg/vacuum/vacuum.go

```diff
@@ -46,15 +47,31 @@ func init() {
 }
 
 const (
-	sqlAcquireLock      = "SELECT _prom_catalog.lock_for_vacuum_engine()"
-	sqlListChunks       = "SELECT format('%I.%I', schema_name, table_name) FROM _ps_catalog.chunks_to_freeze WHERE coalesce(last_vacuum, '-infinity'::timestamptz) < now() - interval '15 minutes' LIMIT 1000"
-	sqlVacuumFmt        = "VACUUM (FREEZE, ANALYZE) %s"
-	sqlReleaseLock      = "SELECT _prom_catalog.unlock_for_vacuum_engine()"
-	delay               = 10 * time.Second
-	defaultDisable      = false
-	defaultRunFrequency = 10 * time.Minute
-	defaultParallelism  = 4
-	minParallelism      = 1
+	sqlAcquireLock       = "SELECT _prom_catalog.lock_for_vacuum_engine()"
+	sqlVacuumFmt         = "VACUUM (FREEZE, ANALYZE) %s"
+	sqlReleaseLock       = "SELECT _prom_catalog.unlock_for_vacuum_engine()"
+	sqlGetVacuumSettings = `
+	SELECT 
+		max(s.setting::bigint) filter (where s.name = 'vacuum_freeze_min_age') as vacuum_freeze_min_age,
+		max(s.setting::bigint) filter (where s.name = 'autovacuum_freeze_max_age') as autovacuum_freeze_max_age
+	FROM pg_catalog.pg_settings s
+	WHERE s.name IN ('vacuum_freeze_min_age', 'autovacuum_freeze_max_age')
+	`
+	sqlListChunks = `
+	SELECT 
+		format('%I.%I', schema_name, table_name) AS schema_table,
+		pg_stat_get_autovacuum_count(format('%I.%I', schema_name, table_name)::regclass::oid) AS autovacuum_count,
+		pg_catalog.age(relfrozenxid) as age
+	FROM _ps_catalog.chunks_to_freeze 
+	WHERE coalesce(last_vacuum, '-infinity'::timestamptz) < now() - interval '15 minutes' 
+	AND pg_catalog.age(relfrozenxid) > (SELECT setting::bigint FROM pg_settings WHERE name = 'vacuum_freeze_min_age')
+	LIMIT 1000`
+	sqlGetAutovacuumCount = "SELECT pg_catalog.pg_stat_get_autovacuum_count($1::regclass::oid)"
+	delay                 = 10 * time.Second
```

Comment:
Nit: maybe worth adding `//` comments just explaining what is it used for

Reply:
https://github.com/timescale/promscale/pull/1761/commits/eb5586f2da3d694d01800342598892dfa5d89d44